### DISCUSSION TITLE
Issue making a dependency on a dbt model

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#18336] [Q&A] [answered]
I am trying to use a dbt model as an input for a Python asset. When I test this out I get the following error.

![image (1)](https://github.com/dagster-io/dagster/assets/19783112/719e4e3d-0c16-4d6b-b858-72f419bc32db)

![image (2)](https://github.com/dagster-io/dagster/assets/19783112/bc9ec171-0754-4c9e-865b-e9bc5b5b928b)

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1701192536528649?thread_ts=1701192536.528649&cid=C01U954MEER).*

**Comment by smackesey** [comment]
cc @rexledesma is something simple missing here about how to use assets downstream of DBT assets?

**Reply by rexledesma** [reply, marked as answer]
Seems like the IO manager for the upstream `base_consumer_offer_generated_lat_long` has not been configured.

This should be configured to fetch the dbt model from the data warehouse, rather than using the `fs_io_manager` to load it from the filesystem (where it doesn't exist).
